Patch by Zarafa <http://www.zarafa.com/> - special characters break the quoted-printable encoding when
they are typed after a ?. This fix also encodes ?-characters, so we can't break the encoding with this
trick.

--- libvmime-0.7.1/src/encoderQP.cpp					2008-03-10 13:57:05.055163691 +0100
+++ libvmime-0.7.1/src/encoderQP.cpp.quoted-printable-encode-question	2008-03-10 13:51:53.071163435 +0100
@@ -249,7 +249,7 @@
 		default:
 		{
 			//if ((c >= 33 && c <= 60) || (c >= 62 && c <= 126))
-			if (c >= 33 && c <= 126 && c != 61)
+			if (c >= 33 && c <= 126 && c != 61 && c!= 63)
 			{
 				outBuffer[outBufferPos++] = c;
 				++curCol;
